sorted_words.rb

```diff
@@ -1 +1,21 @@
-# Write a program called sorted_words.rb.  It should prompt the user for words and add each to an array.  The user should be able to add as many words as they like, until they just hit enter to return a blank word. Then sort the array using the sort method and print it out.
\ No newline at end of file
+# Write a program called sorted_words.rb.  It should prompt the user for words and add each to an array.  The user should be able to add as many words as they like, until they just hit enter to return a blank word. Then sort the array using the sort method and print it out.
+
+def sorted_words
+    words = []
+    puts "Enter a word. "
+    word = gets.chomp
+    words.push(word)
```

Comment:
Is there a particular reason this is duplicated outside the loop?

Reply:
I had originally initiated this with a while loop, and I needed a user-submitted word with a length > 0 to initiate the while loop as that was my condition. 

I then refactored this into a loop but failed to remove these lines of code.